### ISSUE TITLE
Add support for brew command not found

### ISSUE THREAD

**Issue by luisdavim** [nvbn/thefuck#941]
can you add support for https://github.com/Homebrew/homebrew-command-not-found ?

**Comment by jpninanjohn** [comment]
This already seems to be done.

**Comment by scorphus** [comment]
Yes, we have a rule for unknown brew commands: [thefuck/rules/brew_unknown_command.py](https://github.com/nvbn/thefuck/blob/master/thefuck/rules/brew_unknown_command.py)

This issue refers to something else: https://github.com/Homebrew/homebrew-command-not-found

Would you be willing to contribute, @jpninanjohn?

**Comment by aaditkamat** [comment]
@scorphus, I wish to take this up

**Comment by scorphus** [comment]
Go for it, @aaditkamat! 🎃

**Comment by aaditkamat** [comment]
@scorphus, can the following lines be put in `utils.py`?

https://github.com/nvbn/thefuck/blob/4f165bf6dfbe14e0196c787d0a7c328803d960bd/thefuck/rules/brew_unknown_command.py#L13-L41

I feel that this code would be reused in this rule because you want to check whether the command the user typed is a valid Homebrew command before suggesting him to install it.

**Comment by scorphus** [comment]
Hey, @aaditkamat! Glad to know you're on it! 💪

You can move them to [`thefuck/specific/brew.py`](https://github.com/nvbn/thefuck/blob/master/thefuck/specific/brew.py).
